### PR TITLE
Oseibert os 5949 cleanups

### PR DESCRIPTION
**Pull request by oseiberts11**
Some small cleanups:
- added Python 3.8 tests
- inserted return statements,
- improved comments and variable names
- made log message more useful

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Maintainer: Dennis Kuhn <d.kuhn@syseleven.de>
+Section: python
+Priority: optional
+Build-Depends: debhelper (>= 9), 
+               python3-setuptools (>= 0.6b3), 
+               python3-requests,
+               python3-consul,
+               python3-all,
+               python3-setuptools,
+               dh-systemd,
```
Comment (oseiberts11) [inline debian/control]: Systemd???

File: debian/rules [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/make -f
+
+export PBR_VERSION=0.1.0
```
Comment (oseiberts11) [inline debian/rules]: Why do we have this here? Do we need to keep this up to date or is it unneeded?

File: debian/changelog [hunk 3]
```diff
@@ -0,0 +1,48 @@
+consul-lib (0.1.5-0+syseleven1) xenial; urgency=medium
+
+  * Version update
```
Comment (stefreak) [inline debian/changelog]: I believe you need to add a release, or do you want to add it in another MR?

File: debian/control (hunk 1 above)
Comment (stefreak) [reply debian/control]: Yeah agreeed, this is probably not necessary :)

**Pull request merged**